src/test/java/uk/gov/justice/laa/crime/cfecrime/steps/OutputData.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.laa.crime.cfecrime.steps;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OutputData {
+    Outcome initialMeansTest;
+    Outcome fullMeansTest;
+}
```

Comment:
There's only really 1 outcome - its the full outcome if it was done, and the init one if not. I agree the crime docs are weird - init === gross income test, full === disposable income test